### ISSUE TITLE
The value of loss_ins and accuracy  is always 0

### ISSUE THREAD

**Issue by vtHuo** [cardwing/Codes-for-Lane-Detection#40]
Thank you for your work, I use the latest code, and have not made any changes, but the loss and accuracy of the training has been 0, some details are not noticed or caused by the TensorFlow version? I am using 1.10.0. Thank you!
![screenshot from 2019-01-15 14-05-15](https://user-images.githubusercontent.com/40886092/51171615-5477da00-18ec-11e9-8cf6-3ecee54159ef.png)

**Comment by cardwing** [comment]
@vtHuo, I have checked the latest code and the code runs normally in my local server. The following is the snapshot of the training process. I think the problem may exist in the input labels. You can check if the input labels and images are read properly.

![progress](https://user-images.githubusercontent.com/23656119/51175296-dc61e200-18f4-11e9-8fcb-f83731f0ed6a.png)
